### ISSUE TITLE
Equivalent of `{ strictSSL: false }`?

### ISSUE THREAD

**Issue by indexzero** [unshiftio/requests#2]
Was reading the code and it doesn't look like this exists. Not a huge problem, just trying to understand if this is even possible in the browser. My gut tells me not since it would bypass the security settings of the browser itself if users had not made the security exception themselves beforehand.

**Comment by 3rd-Eden** [comment]
That is correct, there is no way of forcing browsers to accept self signed certificates, but once you manually accepted one it should be allowed forever. The only thing we can do is properly handle errors that are a result of this. I don't know if this module does that yet, i'll setup a test and verify this in a bit.
